bindings/matrix-sdk-ffi/src/client.rs

```diff
@@ -243,6 +244,12 @@ impl Client {
 #[uniffi::export(async_runtime = "tokio")]
 impl Client {
     /// Login using a username and password.
+
+    /// # Deprecated
+    /// This method has been replaced by the methods [Client::login_token],
```

Comment:
The right syntax so Rustdoc can create links would be the following:

```suggestion
    /// This method has been replaced by the methods [`Client::login_token`],
```

Can you update this in the entire patch, please?